tests_metricflow/snapshots/test_engine_specific_rendering.py/SqlPlan/BigQuery/test_add_time_expr__plan0.sql

```diff
@@ -0,0 +1,10 @@
+test_name: test_add_time_expr
```

Comment:
Resolved this in https://github.com/dbt-labs/metricflow/pull/1580, so these should go away if you regenerate the snapshots.